src/backend/marsha/core/api.py

```diff
@@ -280,6 +280,7 @@ def get_serializer_context(self):
         if isinstance(user, TokenUser):
             context["roles"] = user.token.payload.get("roles", [])
             context["user_id"] = user.token.payload.get("user_id", None)
+            context["session_id"] = user.token.payload.get("session_id", None)
```

Comment:
why not update the context with the whole token payload? :sweat_smile:

Reply:
Good idea :-)